nagios/check.py

```diff
@@ -200,7 +225,7 @@ def __init__(self, log_path, file_template, logger, hostname, event_func, gauge_
             self.compile_file_template(file_template)
 
         self.tail = TailFile(self.log, self.log_path, self._parse_line)
-        self.gen = self.tail.tail(line_by_line=False, move_end=True)
+        self.gen = self.tail.tail(line_by_line=False, move_end=False)
```

Comment:
I think this should be reverted. This we used for testing on the EC2 instance with a running StackState Agent, correct?

Reply:
yeah true.